raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql

```diff
@@ -0,0 +1,7 @@
+-- Drop RB instrument tables
+DROP TABLE IF EXISTS `aosi`;
+DROP TABLE IF EXISTS `bmi`;
+DROP TABLE IF EXISTS `medical_history`;
+DROP TABLE IF EXISTS `mri_parameter_form`;
+DROP TABLE IF EXISTS `radiology_review`;
+DROP TABLE IF EXISTS `testtest`;
```

Comment:
testtest was removed from the creation instructions so it can be removed from here as well, only travis will use it

Reply:
Done in the next commit!